backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Migration/v0.08-authorization-admin/01-setup-procedures.sql

```diff
@@ -0,0 +1,30 @@
+--Procedure: delegation.get_all_offereddelegations
+CREATE OR REPLACE FUNCTION delegation.get_all_offereddelegations(
+	_offeredbypartyid integer,
+	_resourcetype character varying)
+    RETURNS SETOF delegation.delegationchanges 
+    LANGUAGE 'sql'
+    COST 100
+    STABLE PARALLEL SAFE 
+    ROWS 1000
+
+AS $BODY$
+	  SELECT
+	 delegationchangeid,
+    delegationChangeType,
+    altinnappid,
+    offeredbypartyid,
+    coveredbypartyid,
+    coveredbyuserid,
+    performedbyuserid,
+    blobstoragepolicypath,
+    blobstorageversionid,
+    created,
+	resourceid,
+	resourcetype
+	  FROM delegation.delegationChanges
+	  WHERE
```

Comment:
This will now also select all Revoke delegation changes. In order to only find valid active delegations it will have to group by ResourceId, OfferedByPartyId, CoveredByPartyId and CoveredByUserId and select the newest delegation change for each group. 
If the newest delegation change is a RevokeLast event it can be discarded but that might be easier left up to code.
See the existing get_all_current_changes_offeredbypartyid_only function which does similar lookup but with altinnappid.